### ISSUE TITLE
Display Plugin Name on Widget in Layout Editor

### ISSUE THREAD

**Issue by gsf00001** [SocialEngine/phpv4-feature-requests#128]
What is the feature?
-------------------------------------------
Once a widget is copied from the Available Blocks Section into the Layout, there's no way of knowing which Plugin it belongs to.  

After selecting a Widget in Layout Editor, display the name of the associated Plugin (or Blocks Heading Section) this widget is from (Dev name would be handy as well).  There are many times I have to search through dozens of widgets to determine where this particular widget is from (which often requires copying numerous blocks to the Layout in order to compare what the widget looks like (i.e. settings) to see which one the initial Widget actually is).

Thoughts?

**Comment by DonnaScriptTechs** [comment]
Yes I've done the same on my site with the exception of a tag widget as there isn't one. Good idea.
